packages/trip-details/src/index.tsx

```diff
@@ -0,0 +1,232 @@
+import coreUtils from "@opentripplanner/core-utils";
+import moment from "moment";
+import React, { ReactElement } from "react";
+import { FormattedMessage, FormattedNumber } from "react-intl";
+import { CalendarAlt, Heartbeat, MoneyBillAlt } from "styled-icons/fa-solid";
+
+import * as Styled from "./styled";
+import TripDetail from "./trip-detail";
+import { CaloriesDetailsProps, TripDetailsProps } from "./types";
+
+// Load the default messages.
+import defaultMessages from "../i18n/en-US.yml";
+
+const MESSAGE_ID_PREFIX = "otpUi.TripDetails.";
+const messageIds = {
+  calories: `${MESSAGE_ID_PREFIX}calories`,
+  caloriesDescription: `${MESSAGE_ID_PREFIX}caloriesDescription`,
+  departure: `${MESSAGE_ID_PREFIX}departure`,
+  title: `${MESSAGE_ID_PREFIX}title`,
+  tncFare: `${MESSAGE_ID_PREFIX}tncFare`,
+  transitFare: `${MESSAGE_ID_PREFIX}transitFare`
+};
```

Comment:
This is creating a fair amount of duplication and long-term need to update this object every time a new message is created. Perhaps it'd be better to make a function that assists as follows:

```suggestion
function makeMessageId (suffix: string) {
  return `otpUi.TripDetails.${suffix}`;
}
```

Reply:
I ended up using a slightly different approach in https://github.com/opentripplanner/otp-ui/pull/281/commits/21beebcf571549e35ac87db6072146d47d737417, but that big block is gone.